<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FormTemperatureHistory</class>
 <widget class="QWidget" name="FormTemperatureHistory">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1727</width>
    <height>692</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="QListView" name="listView_vehicles">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="maximumSize">
      <size>
       <width>200</width>
       <height>16777215</height>
      </size>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QGridLayout" name="gridLayout_2">
       <property name="spacing">
        <number>2</number>
       </property>
       <item row="0" column="1">
        <widget class="QPushButton" name="pb_historyToday">
         <property name="text">
          <string>Dzisiaj</string>
         </property>
        </widget>
       </item>
       <item row="0" column="3">
        <widget class="QCheckBox" name="cb_autoReset">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="text">
          <string>Auto Reset</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="0" column="2">
        <spacer name="horizontalSpacer_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="0" column="4">
        <widget class="QPushButton" name="pb_historyReset">
         <property name="text">
          <string>Reset</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QDateEdit" name="dateEdit">
         <property name="wrapping">
          <bool>false</bool>
         </property>
         <property name="buttonSymbols">
          <enum>QAbstractSpinBox::UpDownArrows</enum>
         </property>
         <property name="accelerated">
          <bool>false</bool>
         </property>
         <property name="showGroupSeparator" stdset="0">
          <bool>false</bool>
         </property>
         <property name="minimumDateTime">
          <datetime>
           <hour>2</hour>
           <minute>0</minute>
           <second>0</second>
           <year>2023</year>
           <month>9</month>
           <day>30</day>
          </datetime>
         </property>
         <property name="displayFormat">
          <string>dddd dd MMMM yyyy</string>
         </property>
         <property name="calendarPopup">
          <bool>true</bool>
         </property>
         <property name="timeSpec">
          <enum>Qt::LocalTime</enum>
         </property>
         <property name="date">
          <date>
           <year>2023</year>
           <month>12</month>
           <day>17</day>
          </date>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QWidget" name="wg_chart" native="true">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
